src/OrganisationRegistry.Projections.Delegations/Projections/PersonMandateListProjection.cs

```diff
@@ -175,6 +177,42 @@ public async Task Handle(DbConnection dbConnection, DbTransaction dbTransaction,
             }
         }
 
+        public async Task Handle(DbConnection dbConnection, DbTransaction dbTransaction, IEnvelope<OrganisationCoupledWithKbo> message)
+        {
+            using (var context = _contextFactory().Value)
+            {
+                var organisationNames = context.PersonMandateList.Where(item => item.OrganisationId == message.Body.OrganisationId);
+
+                foreach (var delegationListItem in organisationNames)
+                    delegationListItem.OrganisationName = message.Body.Name;
+
+                var bodyOrganisationNames = context.PersonMandateList.Where(item => item.BodyOrganisationId == message.Body.OrganisationId);
+
+                foreach (var delegationListItem in bodyOrganisationNames)
+                    delegationListItem.BodyOrganisationName = message.Body.Name;
+
+                context.SaveChanges();
+            }
+        }
+
+        public async Task Handle(DbConnection dbConnection, DbTransaction dbTransaction, IEnvelope<OrganisationCouplingWithKboCancelled> message)
+        {
+            using (var context = _contextFactory().Value)
```

Comment:
Why do the other changes in this PR have `using (var context = ContextFactory.CreateTransactional(dbConnection, dbTransaction))` and this one has `using (var context = _contextFactory().Value)`?

Should they all be a mix of both? `_contextFactory().Value.CreateTransactional(...`?

Reply:
`PersonMandateListProjection` seems to be a `BaseProjection` instead of a `Projection` (and the only one still). Not sure when that happened. In the context of this PR it's irrelevant, but we might need to look into that in time.